calc@patapon.info/metadata.json

```diff
@@ -1,7 +1,11 @@
 {
- "shell-version": ["3.12"],
- "uuid": "calc@patapon.info",
- "name": "Calculator",
- "description": "A simple calculator in the search overview",
- "url": "https://github.com/eonpatapon/gnome-shell-extension-calc"
+    "uuid": "calc@patapon.info",
+    "name": "Calculator",
+    "description": "A simple calculator in the search overview",
+    "url": "https://github.com/eonpatapon/gnome-shell-extension-calc",
+    "shell-version": [
+        "3.26",
+        "3.28"
+    ],
+    "version": 14
```

Comment:
I've extracted this _"14"_ from the versions available in: https://extensions.gnome.org/extension/111/calculator/
Indeed, the latest published one is version _"13"_.